packages/react-focus/src/components/FocusZone/FocusZone.EventHandler.test.tsx

```diff
@@ -1,92 +1,145 @@
-import * as ReactDOM from 'react-dom';
 import * as React from 'react';
+import { fireEvent, render } from '@testing-library/react';
+import { KeyCodes } from '@fluentui/utilities';
 import { FocusZone } from './FocusZone';
 
-type EventHandler = {
-  listener: EventListenerOrEventListenerObject;
-  options?: boolean | AddEventListenerOptions;
-};
-
-function optionsEqual(
-  a: boolean | AddEventListenerOptions | undefined,
-  b: boolean | AddEventListenerOptions | undefined,
-) {
-  if (typeof a !== 'object' || typeof b !== 'object') {
-    return a === b;
-  }
-
-  return a.once === b.once && a.passive === b.passive && a.capture === b.capture;
-}
-
-function handlersEqual(a: EventHandler, b: EventHandler) {
-  return a.listener === b.listener && optionsEqual(a.options, b.options);
-}
-
 // HEADS UP: this test is intentionally in a separate file aside from the rest of FocusZone tests
 // As it is testing ref counting on a global `window` object it would interfere with other FocusZone tests
 // which use ReactTestUtils.renderIntoDocument() which renders FocusZone into a detached DOM node and never unmounts.
```

Comment:
This comment appears to no longer be correct. Can it be updated or deleted?

Reply:
Yup, I've just deleted it.